libcudacxx/include/cuda/stream_ref

```diff
@@ -9,7 +9,7 @@
 //===----------------------------------------------------------------------===//
 
 #ifndef _CUDA_STREAM_REF
-#define _CUDA_STREAM_REF
+#  define _CUDA_STREAM_REF
```

Comment:
Do we need to indent all these macros? I would prefer to leave those unchanged in this PR.